contrib/completion/bash/docker

```diff
@@ -138,6 +138,10 @@ __docker_containers_and_images() {
 	COMPREPLY+=( "${containers[@]}" )
 }
 
+__docker_networks() {
+	COMPREPLY=( $(compgen -W "$(__docker_q network ls | awk 'NR>1 {print $2}')" -- "$cur") )
```

Comment:
Can we also try to complete network IDs in addition to names? It's useful and is consistent with what we do for containers (we complete on both IDs and names).